KomodoPlatform/komodo-docs-mdx
# Update makerbot `price_url` param from string to list

Ref: https://github.com/KomodoPlatform/komodo-defi-framework/pull/2027

To test:
- Using latest dev branch mm2 (kdf) binary, activate KMD and LTC.
- Use the example in doc to start the makerbot. If KMD/LTC wallets are funded, set a reasonable percentage in case a trade intiates. 
- Check the logs. At each bot price refresh loop, it should indicate where pricing info was sourced from.
